lib/cylc/config.py

```diff
@@ -2103,7 +2103,7 @@ def load_graph(self):
                                 f"ERROR, "
                                 f"xtrigger function not callable: "
                                 f"{xtrig.func_name}")
-                    except ModuleNotFoundError:
+                    except (ModuleNotFoundError, AttributeError):
```

Comment:
This is where I think we need to catch the other error, to prevent the suite of hanging?

Reply:
This should go in, anyway.. However, the hang happened in the `xtrigger_mgr.py` when the `ctx.out` is `None` and then caused an error when trying to load this as JSON.

Reply:
:+1: after I went for lunch I realized this would probably just fail in the suite config validation/load. But agree that this should be added.